packages/api/src/settings/entities.ts

```diff
@@ -39,8 +39,6 @@ export function getGenericSettingValueRealized(
     normalizedValue = this.value
       .split(',')
       .map((v: string) => Number.parseInt(v.trim()));
-  } else if (this.type === 'Image') {
-    normalizedValue = `${process.env.SERVER_URL as string}/${this.value}`;
   } else {
```

Comment:
I think it is preferable to define the "realized" image url via the server, and just load the specified url in the client.

It should be under the server's control where images are hosted, and under what path.  The client shouldn't need to know anything about our image hosting -- just the url to fetch.

That allows us to make sure every client is accessing images from the proper place in case anything needs to be moved down the road.